src/cephadm/cephadm.py

```diff
@@ -4692,6 +4696,34 @@ def command_list_images(ctx: CephadmContext) -> None:
     # print default images
     cp_obj.write(sys.stdout)
 
+
+def update_service_for_daemon(ctx: CephadmContext,
+                              available_daemons: list,
+                              update_daemon: list) -> None:
```

Comment:
nit: the name of this variable doesn't align with its type, if it's a a list then we should be using `updated_daemons`